eox_core/cms/static/js/index.jsx

```diff
@@ -58,3 +67,9 @@ CourseManagement.propTypes = {
     request_timeout_value: PropTypes.number,
   }),
 };
+
+CourseManagement.propTypes = {
+  courseRerun: PropTypes.shape({
+    request_timeout_value: PropTypes.number,
```

Comment:
I think we should use a standard to define attributes or variables. Whether I understand we must define a style guide for this,  maybe camelcase works better in this context. What do you think @amalbas ???

Reply:
Here is the only part where the props variables have underscored names because they came from of the html template and they are passed as Python context variables, could be amend to camel case names, but keeping those names like so, i think we have more separated logic between props passed from the template and child components props. @diegomillan

Reply:
ok @Squirrel18